### PR TITLE
chore: review done

### LINKED ISSUE #23: "Post on Behalf of Users" Review Adjustments - I
> There's a handful of small adjustments I'd like that we can roll into new task(s) after merging in the current pull request:

- [x] Issue titles should be Title Case and no punctuation at the end
- [x] No more block quote for the message extracted from telegram (we ignore blockquotes for comment incentives. Just write with normal text.
- [x] "[ View Conversation Context ]" to "[ View on Telegram ]"

_Originally posted by @pavlovcik in https://github.com/ubiquity/telegram-ubiquibot/pull/22#pullrequestreview-1618850334_

### PR DESCRIPTION
**Pull request by seprintour**
Resolves #23 
QA: https://github.com/Seprintour-Test/test/issues/42

### CODE REVIEW

**Comment by 0x4007** [comment]
Can you post QA link

File: src/helpers/github.ts [hunk 1]
```diff
@@ -103,6 +103,12 @@ export const createIssue = async (
     const apiUrl = `${GITHUB_API_URL}/repos/${organization}/${repository}/issues`;
 
     const timeCapitalized = capitalizeWords(timeEstimate);
+    let titleCapitalized = capitalizeWords(issueTitle);
+
+    // remove punctuation at the end of the titleCapitalized if it exists
+    if (titleCapitalized[titleCapitalized.length - 1].match(/[.,\/#!$%\^&\*;:{}=\-_`~()]/g)) {
+      titleCapitalized = titleCapitalized.slice(0, -1);
```
Comment (0x4007) [inline src/helpers/github.ts:110]: It punctuates 100% of the time?

Comment (seprintour) [reply src/helpers/github.ts:110]: I don't know, that's why I checked if the end is a punctuation

**Comment by seprintour** [comment]
> Can you post QA link

https://github.com/Seprintour-Test/test/issues/42

**Review by pavlovcik — APPROVED** [review]

**Pull request merged**